### ISSUE TITLE
Reader support for custom artwork?

### ISSUE THREAD

**Issue by peripheralist** [zxing-js/library#470]
I've created some generative artwork that maps to a 40 character hexadecimal string. I'd like to adapt zxing to be able to read it, and would love some help from someone who's more familiar with the library than I am.

Let me know if you're interested!

![image](https://user-images.githubusercontent.com/79433522/131436955-023dbb9b-63a9-48fc-bc79-d3b56dcb1e88.png)

**Comment by Pfed-prog** [comment]
I've used https://www.prepostseo.com/image-to-text
Is the text in any way similar to `Teo
1 1592 7-1
4OINT
J
OY
mant`

**Comment by gabrielmayers** [comment]
https://www.prepostseo.com/image-to-text worked for u @peripheralist?

**Comment by peripheralist** [comment]
no, that tool looks like it's for reading images _of text_ (like a picture of a page in a book). the images i shared need a custom algorithm to be decoded
